src/app/components/layout-sidebar/layout-sidebar-demo.component.ts

```diff
@@ -5,5 +5,19 @@ import { Component, OnInit } from '@angular/core';
   templateUrl: './layout-sidebar-demo.component.html'
 })
 export class StacheLayoutSidebarDemoComponent implements OnInit {
+  public routes: any[] = [
```

Comment:
Ah, that's on the demo page... No we could pull in the `StacheNavLink` from the module, but I'd prefer not to require the consumer knowing which type to use for every attribute (different story when developing within the module).